Podcast/DispatchTime+.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  DispatchTime+.swift
+//  Podcast
+//
+//  Created by Mindy Lou on 10/11/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+extension DispatchTime {
+    static func waitFor(milliseconds: Int, completion: @escaping () -> Void) {
```

Comment:
do we need @escaping? we can probably just do () -> ()

Reply:
It's giving me a compiler error if I don't mark it as escaping